cmd/snap/cmd_wait.go

```diff
@@ -118,6 +165,19 @@ func (x *cmdWait) Execute(args []string) error {
 	snapName := string(x.Positional.Snap)
 	confKey := x.Positional.Key
 
+	if snapName == "godot" && confKey == "" {
```

Comment:
Easter egg?

Reply:
What's the impact of this Easter egg on the existing `godot` snap?

Reply:
No impact because wait always needs "snapname" and "confkey", still probably a bit much fun, I removed it.